services/scan/src/build_cast_vote_record.test.ts

```diff
@@ -203,35 +211,43 @@ test('generates a CVR from a completed BMD ballot with write in and overvotes',
   const blankPageTypes = ['BlankPage', 'UnreadablePage'];
   for (const blankPageType of blankPageTypes) {
     expect(
-      buildCastVoteRecord(sheetId, batchId, batchLabel, ballotId, election, [
-        {
-          interpretation: {
-            type: 'InterpretedBmdPage',
-            ballotId,
-            metadata: {
-              locales: { primary: 'en-US' },
-              electionHash: electionDefinition.electionHash,
-              ballotType: BallotType.Standard,
-              ballotStyleId,
-              precinctId,
-              isTestMode: false,
-            },
-            votes: vote(contests, {
-              '1': {
-                id: 'write-in-PIKACHU',
-                name: 'Pikachu',
-                isWriteIn: true,
+      buildCastVoteRecord(
+        sheetId,
+        batchId,
+        batchLabel,
+        ballotId,
+        election,
+        [
+          {
+            interpretation: {
+              type: 'InterpretedBmdPage',
+              ballotId,
+              metadata: {
+                locales: { primary: 'en-US' },
+                electionHash: electionDefinition.electionHash,
+                ballotType: BallotType.Standard,
+                ballotStyleId,
+                precinctId,
+                isTestMode: false,
               },
-              '2': ['21', '22'],
-            }),
+              votes: vote(contests, {
+                '1': {
+                  id: 'write-in-PIKACHU',
+                  name: 'Pikachu',
+                  isWriteIn: true,
+                },
+                '2': ['21', '22'],
+              }),
+            },
           },
-        },
-        {
-          interpretation: {
-            type: blankPageType as 'BlankPage' | 'UnreadablePage',
+          {
+            interpretation: {
+              type: blankPageType as 'BlankPage' | 'UnreadablePage',
```

Comment:
I know this was pre-existing, but you can remove this cast with `const blankPageTypes = ['BlankPage', 'UnreadablePage'] as const;` above.